backend/common/ai_drive.py

```diff
@@ -16,26 +17,45 @@
 from backend.ml.ml_trainer import train_classical_ml_model
 from backend.ml.ml_model_parser import get_object_ml
 
+from backend.aws_helpers.dynamo_db_utils.execution_db import ExecutionDDBUtil, ExecutionData, getOrCreateUserExecutionsData_, updateUserExecutionsData_, createExecutionID
+
 
 def dl_tabular_drive(
-    user_arch,
-    criterion,
-    optimizer_name,
-    problem_type,
-    fileURL,
-    target=None,
-    features=None,
-    default=None,
-    test_size=0.2,
-    epochs=5,
-    shuffle=True,
-    batch_size=20,
-    json_csv_data_str=""
+    user_arch: list,
+    fileURL: str,
```

Comment:
dont forget to consider default dataset case here

Reply:
do we rlly need json_csv_data_str considering thing with s3? thatll be for another iteration, not for this pr

Reply:
The default cases have been considered. json_csv_data was already here before. I don't want tk remove it until we get the S3 thing working